mlir/test/Dialect/Vector/canonicalize.mlir

```diff
@@ -652,24 +652,44 @@ func.func @fold_extract_transpose(
 
 // -----
 
-// CHECK-LABEL: fold_extract_broadcast
+// CHECK-LABEL: fold_extract_broadcast_same_type
 //  CHECK-SAME:   %[[A:.*]]: f32
 //       CHECK:   return %[[A]] : f32
-func.func @fold_extract_broadcast(%a : f32) -> f32 {
+func.func @fold_extract_broadcast_same_type(%a : f32, 
```

Comment:
```suggestion
func.func @fold_extract_broadcast_scalar_f32(%a : f32, 
```
I mostly care about "broadcast_scalar" to clearly distinguish with the example below.

Reply:
Made it more clear.